_data/simple-icons.json

```
@@ -15371,6 +15371,11 @@
             "source": "https://brandfolder.com/portals/vivint",
             "guidelines": "https://s.tiled.co/2wH-ED5/branding-guide"
         },
+        {
+            "title": "vivo",
+            "hex": "000000",
```

```suggestion
            "hex": "415FFF",
```

I agree with @PeterShaggyNoble's observation on the color used.